### ISSUE TITLE
Screen reader user is unable to differentiate current date and previously selected date in the date picker

### ISSUE THREAD

**Issue by gkhadK** [vmware/clarity#4635]
**Clarity Version:** 3.1.1
**WCAG 2.1 Guideline:** 4.1.2 Name, Role, Value 

**Test Environment:**
- Test URL: https://clarity.design/documentation/datepicker
- OS: Windows 10
- User Agents/Assistive Technologies: Firefox 76.0
- Screen reader: NVDA 2020.1

**Steps to reproduce:**
1. Open the test URL.
2. Go to toggle datepicker button and press it.
3. Select any random date from the date picker for current month.
4. Open the datepicker again and observe spoken feedback for non selected dates, selected default and current/actual date.

**Expected result:**
Screen reader user should be able differentiate current date and previously selected date from other dates in that calendar month.

**Actual result:**
Screen reader user cannot distinguish different states of date, whereas same info is communicated to the sighted user visual cues.

**Comment by gnomeontherun** [comment]
Hi there, this is an automated message.

In an effort to dedupe and track accessibility reports consistently, this issue is being closed and tracked internally with our Accessibility team. It will still be triaged and addressed as part of ongoing work.
